packages/drag-drop/src/styled/draggable/StyledDraggable.ts

```diff
@@ -0,0 +1,160 @@
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'draggable';
+
+export interface IStyledDraggableProps extends ThemeProps<DefaultTheme> {
+  focusInset?: boolean;
+  isBare?: boolean;
+  isCompact?: boolean;
+  isDisabled?: boolean;
+  isGrabbed?: boolean;
+  isPlaceholder?: boolean;
+}
+
+export function getDragShadow(theme: IGardenTheme) {
+  const { space, shadows } = theme;
+  const offsetY = `${space.base * 5}px`;
+  const blurRadius = `${space.base * 7}px`;
+  const color = getColor('neutralHue', 600, theme, 0.35) as string;
+
+  return shadows.lg(offsetY, blurRadius, color);
+}
+
+export function getFocusShadow(
+  focusInset: IStyledDraggableProps['focusInset'],
+  baseColor: string,
+  theme: IGardenTheme
+) {
+  return `${focusInset ? 'inset ' : ''}${theme.shadows.md(rgba(baseColor as string, 0.35))}`;
+}
+
+const colorStyles = (props: IStyledDraggableProps) => {
+  const { isBare, isGrabbed, isDisabled, isPlaceholder, focusInset, theme } = props;
+
+  const baseColor = getColor('primaryHue', 600, theme);
+  const dragShadow = getDragShadow(theme);
+  const focusShadow = getFocusShadow(focusInset, baseColor!, theme);
+  const baseBgColor = theme.colors.background;
+
+  let color;
+  let hoverBackgroundColor;
+  let boxShadow;
+  let focusBoxShadow;
+  let borderColor = 'transparent';
+  let backgroundColor = baseBgColor;
+
+  if (isDisabled) {
+    backgroundColor = getColor('neutralHue', 200, theme)!;
+    color = getColor('neutralHue', 400, theme);
+  } else if (isPlaceholder) {
+    backgroundColor = getColor('neutralHue', 800, theme, 0.1)!;
+  } else {
+    color = theme.colors.foreground;
+    borderColor = isBare ? 'transparent' : (getColor('neutralHue', 300, theme) as string);
+    hoverBackgroundColor = isGrabbed ? baseBgColor : rgba(baseColor as string, 0.08);
+    focusBoxShadow = isGrabbed ? `${focusShadow}, ${dragShadow}` : focusShadow;
+    boxShadow = dragShadow;
+  }
+
+  return css`
+    border-color: ${borderColor};
+    background-color: ${backgroundColor};
+    color: ${color};
+
+    ${isGrabbed &&
+    css`
+      box-shadow: ${boxShadow};
+    `}
+
+    &:hover {
+      background-color: ${hoverBackgroundColor};
+    }
+
+    &[data-garden-focus-visible] {
+      box-shadow: ${focusBoxShadow};
+    }
+  `;
+};
+
+const cursorStyles = (props: IStyledDraggableProps) => {
+  let cursor = props.isGrabbed ? 'grabbing' : 'grab';
+
+  if (props.isDisabled || props.isPlaceholder) {
+    cursor = 'default';
+  }
+
+  return css`
+    cursor: ${cursor};
+  `;
+};
+
+const sizeStyles = (props: IStyledDraggableProps) => {
+  const { isCompact, theme } = props;
+
+  /**
+   * 1. Reset margin, e.g. when custom tag includes native margin
+   */
+  return css`
+    margin: 0; /* [1] */
+    border-width: ${theme.borderWidths.sm};
+    border-style: ${theme.borderStyles.solid};
+    border-radius: ${theme.borderRadii.md};
+    padding: ${isCompact ? theme.space.base * 1.25 : theme.space.base * 2.25}px;
```

Comment:
Not sure if this is the right placement for this comment, but `isCompact` styling should have the same left/right | start/end padding/spacing as default i.e. 8px